spec/models/vacancy_spec.rb

```diff
@@ -322,7 +322,7 @@
         create(:vacancy, :expires_tomorrow)
 
         expect(PublishedVacancy.expired_yesterday.count).to eq(1)
-        expect(DraftVacancy.expired_yesterday.count).to eq(1)
```

Comment:
Question: Since this scope lives now in the `PublishedVacancy`. Do we want to remove the `create(:draft_vacancy)` line and this commented line?

Do we have any use of the `#expired_yesterday` for a Draft Vacancy? (I assume not since the scope was moved to `PublishedVacancy`.